### PR TITLE
Integrate Container API changes

### PR DESCRIPTION
**Pull request by Praveengostu**

### CODE REVIEW

**Comment by StrongMonkey** [comment]
👍 let me know when this will get merged. We are building the [cluster provision tool](https://github.com/rancher/kontainer-engine) and would like to integrate IBM driver into our own.

File: api/container/containerv1/subnets.go [hunk 1]
```diff
@@ -25,10 +25,18 @@ type SubnetProperties struct {
...
+	CIDR   string `json:"cidr" binding:"required" description:"The CIDR of the subnet that will be bound to the cluster. Eg.format: 12.34.56.78/90"`
+	VLANID string `json:"vlan_id" binding:"required" description:"The private VLAN where the CIDR exists'"`
+}
+
 //Subnets interface
 type Subnets interface {
 	AddSubnet(clusterName string, subnetID string, target ClusterTargetHeader) error
 	List(target ClusterTargetHeader) ([]Subnet, error)
+	AddClusterUserSubnet(clusterID string, userSubnet UserSubnet, target ClusterTargetHeader) error
+	ListClusterUserSubnets(clusterID string, target ClusterTargetHeader) ([]Vlan, error)
```
Comment (ashishth09) [inline api/container/containerv1/subnets.go]: Is it supposed to return subnets or Vlans?

File: api/container/containerv1/workers.go [hunk 2]
```diff
@@ -8,24 +8,28 @@ import (
...
+	PublicVlan    string `json:"publicVlan"`
+	State         string `json:"state"`
+	Status        string `json:"status"`
+	TargetVersion string `json:"targetVersion"`
 }
 
 //WorkerParam ...
 type WorkerParam struct {
-	Action string
-	Count  int
```
Comment (ashishth09) [inline api/container/containerv1/workers.go]: How does removing this affect backward compatibility?

File: api/container/containerv1/subnets.go (hunk 1 above)
Comment (ashishth09) [reply api/container/containerv1/subnets.go]: https://containers.bluemix.net/swagger-api/#!/clusters/GetClusterUserSubnet shows response type as `string` which is definitely not right. Issue has been created to rectify that.

**Pull request merged**